.ci/pitest.sh

```diff
@@ -90,6 +90,8 @@ pitest-common)
   "DefaultLogger.java.html:<td class='covered'><pre><span  class='survived'>        if (closeError) {</span></pre></td></tr>"
   "DefaultLogger.java.html:<td class='covered'><pre><span  class='survived'>        if (closeInfo) {</span></pre></td></tr>"
   "DefaultLogger.java.html:<td class='covered'><pre><span  class='survived'>        if (severityLevel != SeverityLevel.IGNORE) {</span></pre></td></tr>"
+  "DefaultLogger.java.html:<td class='uncovered'><pre><span  class='survived'>        return errorWriter;</span></pre></td></tr>"
+  "DefaultLogger.java.html:<td class='uncovered'><pre><span  class='survived'>        return infoWriter;</span></pre></td></tr>"
```

Comment:
Needed for flushing streams while overriding some methods. Getter-setter for `infoWriter` and `errorWriter` have been added to flush the streams in the anonymous inner class. The anonymous inner class is needed to get rid of `audit started` and `audit done` messages.